documentation/src/main/asciidoc/topics/proc_query_queries_by_keys.adoc

```diff
@@ -0,0 +1,49 @@
+[id='queries_by_keys_{context}']
+= Queries by keys
+
+If a key of a cache entry has an indexed type, it is possible to index the key fields as well the the value fields,
+so that also the formers could be used in the `Ickle` queries.
```

Comment:
Former cannot be pluralized. Reword as:
Use indexed types as keys to query caches by key using Ickle.